homeassistant/components/xiaomi_aqara/binary_sensor.py

```diff
@@ -181,6 +182,12 @@ def extra_state_attributes(self):
         attrs.update(super().extra_state_attributes)
         return attrs
 
+    async def async_added_to_hass(self) -> None:
+        """Handle entity which will be added."""
+        await super().async_added_to_hass()
+        self._state = False
+        self.async_schedule_update_ha_state()
```

Comment:
This isn't needed. State will be updated once at the end of entity addition.
```suggestion
```

Reply:
done